src/views/SpaceView.vue

```diff
@@ -6,6 +6,7 @@ const router = useRouter();
 const { domain } = useApp();
 const aliasedSpace = aliases[domain] || aliases[route.params.key as string];
 const { loadExtendedSpace, extendedSpaces } = useExtendedSpaces();
+const forceShow = ref(false);
```

Comment:
```suggestion
const showFlaggedWarning = ref(false);
```

forceShow can mean a lot